www/pages/error/ErrorPage.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+
+import { AppTree } from '../../shared/tests';
+import ErrorPage from './ErrorPage';
+
+describe('ErrorPage', () => {
```

Comment:
This grouping is unnecessary as this test file is dedicated for the component and we are only testing it.

Reply:
I see it now. Should I also remove the `describe('getInitialProps')` grouping? That one is useful, I think.

Reply:
Yep you should keep that one.